src/things_mcp/applescript_bridge.py

```
@@ -113,6 +160,187 @@ def add_todo_direct(title: str, notes: Optional[str] = None, when: Optional[str]
+    logger.debug(f"Executing AppleScript: {script}")
+
+    result = run_applescript(script)
+    if result:
+        logger.info(f"Successfully created project with ID: {result}")
+        return result
+    else:
+        logger.error("Failed to create project")
+        return False
+
+
+def update_project_direct(id: str, title: Optional[str] = None, notes: Optional[str] = None,
```

Ruff (0.15.6)

[warning] Too many branches (25 > 12)

(PLR0912)